hasura/metadata/databases/default/tables/public_invite.yaml

```diff
@@ -21,7 +21,32 @@ select_permissions:
         - email
         - expires_at
       filter: {}
+  - role: user
+    permission:
+      columns:
+        - email
+        - expires_at
+        - id
+        - team_id
+      filter:
+        team:
+          memberships:
+            _and:
+              - team_id:
+                  _eq: X-Hasura-Team-Id
+              - role:
+                  _eq: OWNER
```

Comment:
need to add user id here too

Reply:
@andy-t-wang Updated for select and delete permissions: 


https://github.com/worldcoin/developer-portal/assets/89008845/d01fd421-f985-4c3a-88a3-1083935a4b79